src/main/webapp/WEB-INF/jsp/courseConfiguration.jsp

```diff
@@ -218,7 +218,28 @@
         </div>
     </div>
 
-    <div class="confirmation-modal modal fade in" id = "pushModal">
+    <div class="confirmation-modal modal fade in" id = "sectionSelect">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title">Select which Sections will have assignments:</h4>
+                </div>
+                <div class="modal-body">
+                        <c:forEach items="${courseConfigurationForm.allSections}" var="section">
+                            <hr width="0" size="0">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "width" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "size" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)